File: 002+003/003.cpp
//nhap tam duong tron
//nhap ban kinh
//nhap toa do M
//kiem tra |MC| voi R

#include <iostream>
#include <cmath>
#include "vector.h"

using namespace std;

int main() {
	double xC, yC, R, xM, yM, l;

	cout << "Nhap vao toa do tam C(xC, yC)? ";
	cin >> xC >> yC;

	cout << "Nhap ban kinh R? ";
	cin >> R;

	cout << "Nhap vao toa do M(xM, yM)? ";
	cin >> xM >> yM;

	l = do_dai_vecto(xC, yC, xM, yM);

	if (l < R) {
		cout << "M nam trong C()\n";	
	} else if (l == R) {
		cout << "M nam tren C()\n";
	} else {
		cout << "M nam ngoai C()\n";
	}
	return 0;
}

File: 002+003/003.js
const readline = require('readline');
const do_dai_vecto = require('./vector');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Nhap toa do tam C (xC, yC): ', (inputC) => {
    const [xC, yC] = inputC.split(' ').map(Number);

    rl.question('Nhap ban kinh R: ', (inputR) => {
        const R = parseFloat(inputR);

        rl.question('Nhap toa do M (xM, yM): ', (inputM) => {
            const [xM, yM] = inputM.split(' ').map(Number);

            const lengthCM = do_dai_vecto(xC, yC, xM, yM);
            
            if (lengthCM < R) {
                console.log("M nam trong C()");
            } else if (lengthCM === R) {
                console.log("M nam tren C()");
            } else {
                console.log("M nam ngoai C()");
            }

            rl.close();
        });
    });
});